### PR TITLE
[15.0][ADD] sale_fully_invoiced

### PR DESCRIPTION
**Pull request by AaronHForgeFlow**
Split from https://github.com/OCA/sale-workflow/pull/2306 according to this comment: https://github.com/OCA/sale-workflow/pull/2306#pullrequestreview-1218920523

The standard field invoice_status show information about the status invoices related
to a sales Order. However, it can be "invoiced" when the invoices are still in draft.
Many times that makes the difference to consider the Sales Order
as closed or not. Because of this, this module introduce two new fields that  will show
that the Sales Order is fully delivered and Fully Invoiced, taking the state into
account.

cc @ForgeFlow

### CODE REVIEW

**Review by JuanyDForgeflow — APPROVED** [review]
Functional Review LGTM

File: sale_fully_invoiced/models/sale.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        ],
+        string="Posted Invoice Status",
+        compute="_compute_invoice_status_validated",
+        store=True,
+        readonly=True,
+        copy=False,
+        default="no",
+    )
+
+    @api.depends("invoice_status")
```
Comment (em230418) [inline sale_fully_invoiced/models/sale.py]: Missing "invoice_ids.state" in "depends".

Comment (AaronHForgeFlow) [reply sale_fully_invoiced/models/sale.py]: Good catch. Fixed.

**Review by MiquelRForgeFlow — APPROVED** [review]
LGTM

File: sale_fully_invoiced/tests/test_sale_fully_invoiced.py [hunk 2]
```diff
@@ -0,0 +1,70 @@
+# Copyright 2022 ForgeFlow, S.L.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+
+from odoo.tests import common, tagged
+
+
+@tagged("-at_install", "post_install")
+class TestSaleFullyDelivered(common.TransactionCase):
+    def setUp(self):
```
Comment (rousseldenis) [inline sale_fully_invoiced/tests/test_sale_fully_invoiced.py]: @AaronHForgeFlow Could you change this to classmethod ?

**Review by rousseldenis — APPROVED** [review]
LGtTM. Minor comment

File: sale_fully_invoiced/tests/test_sale_fully_invoiced.py (hunk 2 above)
Comment (AaronHForgeFlow) [reply sale_fully_invoiced/tests/test_sale_fully_invoiced.py]: @rousseldenis did that :) sorry for the delay

**Pull request merged**